libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -128,6 +128,33 @@ const AP_Param::GroupInfo AP_AHRS::var_info[] = {
     AP_GROUPINFO("EKF_TYPE",  14, AP_AHRS, _ekf_type, 2),
 #endif
 
+    // @Param: ROT_ROLL
```

Comment:
I think a name AHRS_CUSTOM_ROLL would be clearer (most users look only at the var name, not the docs)

Reply:
done